src/sds_data_model/metadata.py

```diff
@@ -1,425 +1,120 @@
-from abc import ABC
-from dataclasses import dataclass, field
-from typing import Any, Dict, List, Optional, TypeVar, Type, Tuple, Union
-from xml.etree.ElementTree import Element, fromstring
-from re import compile
-from requests import get
-from functools import singledispatch
-from typing import List, Union, Tuple
-from xml.etree.ElementTree import Element
+from dataclasses import dataclass
+from typing import Dict, List, TypeVar, Union
 
+from lxml.etree import Element, parse
 
-from sds_data_model.constants import GCO, GMD, GML, GMX, CHARACTER_STRING
+from sds_data_model.constants import TITLE_XPATH, ABSTRACT_XPATH, LINEAGE_XPATH, METADATA_DATE_XPATH, METADATA_LANGUAGE_XPATH, SPATIAL_DATA_SERVICE_TYPE_XPATH,RESOURCE_TYPE_XPATH, FILE_IDENTIFIER_XPATH
```

Comment:
I think the spatial data service type is "dataset or series" but, as you say, there's nothing in the standard or the XML, so you can ignore it.

```suggestion
from sds_data_model.constants import TITLE_XPATH, ABSTRACT_XPATH, LINEAGE_XPATH, METADATA_DATE_XPATH, METADATA_LANGUAGE_XPATH, RESOURCE_TYPE_XPATH, FILE_IDENTIFIER_XPATH
```